package.json

```diff
@@ -100,13 +100,21 @@
       "default": true,
       "order": 7
     },
-    "useFlowParser": {
-      "title": "Use Flow Parser",
-      "description": "Use the [flow](https://github.com/facebook/flow) parser instead of [babylon](https://github.com/babel/babylon).",
+    "jsxBracketSameLine": {
+      "title": "JSX Bracket Same Line",
+      "description": "Put > on the last line in JSX",
```

Comment:
I thought I copied from the prettier docs...but you're right. It should be "Put > on the same line in JSX". Will fix.